DetectorDescription/DDCMS/plugins/dd4hep/DDDefinitions2Objects.cc

```diff
@@ -1760,6 +1863,9 @@ void Converter<DDLDivision>::operator()(xml_h element) const {
     Volume child = parent.divide(childName, static_cast<int>(axesmap.at(axis)), numCopies, startInDeg, widthInDeg);
 
     ns.context()->volumes[childName] = child;
+    // ns.context()->volPtrs[childName] = &(ns.context()->volumes[childName]);
+    // TGeoVolumeMulti objects not needed in volPtrs list, and they cause
+    // crash when IsAssembly() method is called on them.
```

Comment:
@cvuosalo - no need to check if it is an assembly as long as you let dd4hep handle it.